app/models/comment.rb

```diff
@@ -39,15 +39,22 @@ def create_subscription
   def notify(action:, actor:, recipients:)
     case action.to_s
     when 'create'
+      if recipients
+        create_notifications(action: :mention, actor: actor, recipients: recipients)
```

Comment:
Why do we need this extra call? It seems we're duplicating what we used to do with a single line in L43 (pre) and L47 (now).

Reply:
L47 notifies the all the mentioned users in the comment while L58 mentions the subscribers. To implement the described feature, this extra line will have to be added. Note that the `recipients` argument is not used before but it was there because it's used by the other notifiables and the caller would always pass that argument. As for why this was done, I've updated the card to describe the reasoning.